navigator/backend/src/main/scala/com/digitalasset/navigator/model/PartyState.scala

```diff
@@ -4,18 +4,26 @@
 package com.daml.navigator.model
 
 import java.util.concurrent.atomic.AtomicReference
-
 import com.daml.lf.{iface => DamlLfIface}
 import com.daml.ledger.api.refinements.ApiTypes
-import com.daml.navigator.config.UserConfig
+
+import scala.collection.immutable.LazyList
 import scalaz.Tag
 
+import java.net.URLEncoder
+
 case class State(ledger: Ledger, packageRegistry: PackageRegistry)
 
 /** A DA party and its ledger view(s). */
-class PartyState(val config: UserConfig) {
-  val name = config.party
-  val useDatabase = config.useDatabase
+class PartyState(
+    val name: ApiTypes.Party,
+    // A named role specified in our config. Not sure if this is really used?
```

Comment:
```suggestion
    // A named role specified in our config. The role is an arbitrary string chosen by the user. It gets passed to the frontend, which can display a custom theme or custom views depending on the role.
```

... at least in theory, I'm not sure whether it actually works